ArduPlane/GCS_Mavlink.cpp

```diff
@@ -2067,6 +2067,51 @@ void GCS_MAVLINK::handleMessage(mavlink_message_t* msg)
         break;
     }
 
+    case MAVLINK_MSG_ID_SET_POSITION_TARGET_GLOBAL_INT:
+    {
+        mavlink_set_position_target_global_int_t pos_target;
+        mavlink_msg_set_position_target_global_int_decode(msg, &pos_target);
+        const uint16_t alt_mask = 0b0000000000000100; // (alt mask at bit 3)
+            
+        bool msg_valid = true;
+        AP_Mission::Mission_Command cmd = {0};
+
+        if(pos_target.type_mask ^ alt_mask)
```

Comment:
Updated in latest push.  I think I've addresses all coding style concerns.  

For the mask, which is confusing, I've updated how I do it to hopefully be less confusing and added some comments to explain that mavlink is doing things backwards. ;)